### PR TITLE
Refine the schemas for generating classes in the demo for Seed

### PR DESCRIPTION
**Pull request by RomainLofaso**
In relation with https://github.com/criteo/paf-mvp-implementation/pull/94

### CODE REVIEW

File: mvp-spec/json-schemas/post-seed-request.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "post-seed-request",
+    "title": "POST /v1/seed request",
```
Comment (OlivierChirouze) [inline mvp-spec/json-schemas/post-seed-request.json:4]: Should be `POST /paf-proxy/v1/seed request` (missing proxy prefix is confusing, I had to look for implementation to understand)

File: mvp-spec/json-schemas/post-seed-response.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "post-seed-response",
+    "title": "POST /v1/seed request",
```
Comment (OlivierChirouze) [inline mvp-spec/json-schemas/post-seed-response.json:4]: `POST /paf-proxy/v1/seed response` (missing prefix and incorrect "request")

**Review by OlivierChirouze — CHANGES_REQUESTED** [review]

File: mvp-spec/json-schemas/post-seed-request.json (hunk 1 above)
Comment (RomainLofaso) [reply mvp-spec/json-schemas/post-seed-request.json]: Done

File: mvp-spec/json-schemas/post-seed-response.json (hunk 2 above)
Comment (RomainLofaso) [reply mvp-spec/json-schemas/post-seed-response.json]: Done

**Pull request merged**